01-node-tutorial/answers/17-http-stream.js

```diff
@@ -0,0 +1,16 @@
+var http = require('http')
+var fs = require('fs')
```

Comment:
For modern code style, always use `const` unless you need to use `let` to  be able to reassign to the same variable later.